src/app/components/TransactionDialog/index.tsx

```diff
@@ -170,87 +159,33 @@ export const TransactionDialog: React.FC<ITransactionDialogProps> = ({
   );
 };
 
-const getWalletName = wallet => {
-  switch (wallet) {
-    case 'liquality':
-      return 'Liquality';
-    case 'nifty':
-      return 'Nifty';
-    case 'portis':
-      return 'Portis';
-    case 'ledger':
-      return 'Ledger';
-    case 'trezor':
-      return 'Trezor';
-    case 'wallet-connect':
-      return 'Wallet Connect';
+export const getTransactionTitle = (tx: TxStatus, action: string = '') => {
+  switch (tx) {
+    case TxStatus.FAILED:
+      return (
+        <Trans
+          i18nKey={'translations.transactionDialog.pendingUser.failed'}
```

Comment:
Does this work? Because this is the first time I see `i18nKey` passed as a string and I don't think it should work.